example/temperature.cpp

```diff
@@ -0,0 +1,143 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2020 Yves Delley
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include <units/origin.h>
+#include <units/physical/si/base/thermodynamic_temperature.h>
+#include <units/physical/si/prefixes.h>
+#include <units/quantity_io.h>
+#include <units/quantity_point.h>
+#include <units/ratio.h>
+#include <iostream>
+
+using namespace units;
+
+namespace si = units::physical::si;
+
+namespace detail {
+
+struct centikelvin : scaled_unit<ratio(1, 100), si::kelvin> {
+};
+
+}  // namespace detail
+
+struct celsius : alias_unit<si::kelvin, "\u00b0C", no_prefix> {
```

Comment:
Looking at
https://github.com/mpusz/units/blob/017183653d502fcc06a4bb6c7df4e42b6b4c8739/src/include/units/physical/si/prefixes.h#L38
I think this should look something like
```suggestion
struct celsius : alias_unit<si::kelvin, basic_symbol_text{"\u00b0C", "`C"}, no_prefix> {
```
The error in debug mode stems from using an audit contract. I think the contract should always be asserted, since it is cheap and compile-time only.
Replace the `` ` `` with something more appropriate for ASCII-mode only.